tools/doc/html.js

```diff
@@ -152,16 +152,40 @@ function parseLists(input) {
   var state = null;
   var depth = 0;
   var output = [];
+
+  let headingIndex = -1;
+  let heading = null;
+
   output.links = input.links;
-  input.forEach(function(tok) {
+  input.forEach(function(tok, index) {
     if (tok.type === 'code' && tok.text.match(/Stability:.*/g)) {
-      tok.text = parseAPIHeader(tok.text);
-      output.push({ type: 'html', text: tok.text });
+      const stabilityTextRegExp = /(.*:)\s(\d)([\s\S]*)/;
```

Comment:
can probably be a top-level variable, that way we don't need to pass it to `parseAPIHeader`